db/helpers/common.js

```diff
@@ -137,6 +139,17 @@ function createDBLog(inst, eventType, changedKeys, ignoreAttributes) {
 }
 
 /**
+ * A function to see if an object is a room object or not. It returns true
+ * if an object passed has 'type' and 'settings' as its property.
+ * @param  {Object}  obj - An object instance
+ * @returns {Boolean} - returns true if the object has the property
+ * "type" && "settings"
+ */
+function isThisRoom(obj) {
```

Comment:
also, functions which return boolean should be named as positive assertion, so this should be "isRoom"

Reply:
i was just copying the same format that you were using  in https://github.com/salesforce/refocus/blob/master/realtime/utils.js#L37
for subjects and samples